packages/react-ui/src/hooks/useSort.test.ts

```diff
@@ -63,9 +68,84 @@ describe("useSort", () => {
 
     // after new column click, asc
     expect(result.current.sort).toEqual({
+      alwaysSorted: false,
       direction: "asc",
       sortBy: "date",
     })
     expect(result.current.sortQueryString).toEqual("date")
   })
+
+  it("works when alwaysSorted is true", async () => {
+    const { result } = renderHook(() =>
+      useSort({ direction: undefined, sortBy: undefined, alwaysSorted: true }),
+    )
+
+    // initial state
+    expect(result.current.sort).toEqual({
+      direction: undefined,
+      sortBy: undefined,
+      alwaysSorted: true,
+    })
```

Comment:
is this valid? if `alwaysSorted` is true shouldn't `sortBy` and `direction` be defined? we can enforce this with typescript as well but just wondering if there's a case I'm missing/unaware of (not exactly sure on ticket criteria 😅)

Reply:
`alwaysSorted` refers to the action of the sort after the third click when the `direction` equals `desc`. `alwaysSorted' changes the default behavior of the third click from setting `direction`  and `sortBy` to `undefined` to restart the sort from `sortBy` and `direction` set to `asc`.

Hopefully that makes sense 😅

Reply:
got it, makes sense!